src/components/Onboarding/OnboardingCard.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+import { useState, useEffect } from "react";
+import NicknameInput from "@/components/Onboarding/NicknameInput";
+import UserNameInput from "./UserNameInput";
+import { PaperAirplaneIcon } from "@heroicons/react/24/outline";
```

Comment:
The original icon comes from outline/20

Reply:
I recently encountered a problem importing 20/outline. I think we can debug that in the weekly meeting.

Reply:
Turns out I misremembered Heroicons' spec and there is no 20/outline. Sorry for the mistake.